Users/models.py

```diff
@@ -26,5 +34,23 @@ def __str__(self):
 		return self.owner.username
 
 class Friend(models.Model):
-	FriendsList = models.ForeignKey(FriendsList, on_delete=models.CASCADE)
-	friend = models.OneToOneField(User, on_delete=models.CASCADE)
+
+	users = models.ManyToManyField(User, default=defaultUser)
+
+	#NEEDS REVIEW, not sure how to handle on_delete
```

Comment:
You can delete this comment, you handled it correctly. Essentially, you are saying that if the foreign key is deleted from its DB table, then delete it from this DB table, too

Reply:
Ah, alright. Thank you for the insight.